{"indexes":[{"v":{"$numberInt":"2"},"key":{"_id":{"$numberInt":"1"}},"name":"_id_"}],"uuid":"1727951c559b463d89a7f67665aa0c6d","collectionName":"tf.compat.v1.keras.losses.MeanSquaredLogarithmicError","type":"collection"}